pixelfusion/docs-internal-processes
# Feature/api deploy

| Q | A |
|--|--|
| Status | READY |
| Migrations | NO |

## Description
Moved the api endpoint to the public folder, so it can be consumed from the current website.
Instead of adding an extra subdomain.

## Steps to test/reproduce
_Outline the steps to test or reproduce the PR._

1. Pull the changes
1. Run `composer install`
1. Setup your `.env`
1. Run `php -S localhost:8000 -t public`
1. Go to `http://localhost:8000/api`
1. When everything went right you should see a response
